2.3 Standard Payloads Layout.md

```diff
@@ -625,95 +627,96 @@ The Table 2.3.7 describes the dRNG `Beacon` and `CollectiveBeacon` payload's ser
+            <td>The identifier of the dRNG instance.</td>
+        </tr>
+        <tr>
+            <td valign="top">dRNG subpayload <code>oneOf</code></td>
+            <td colspan="2">
+                <details open="true">
+                <summary>TypeBeacon</summary>
+                    <blockquote>
+                        Defines payload data for Beacon payload type.
+                    </blockquote>
+                    <table>
+                        <tr>
+                            <td><b>Name</b></td>
+                            <td><b>Type</b></td>
+                            <td><b>Description</b></td>
+                        </tr>
+                        <tr>
+                            <td>Round</td>
+                            <td>uint64</td>
+                            <td>The round of the current beacon.</td>
+                        </tr>
+                        <tr>
+                            <td>PartialPK</td>
+                            <td>ByteArray[96]</td>
+                            <td>The public key of the issuer.</td>
+                        </tr>
+                        <tr>
+                            <td>PartialSignature</td>
+                            <td>ByteArray[96]</td>
+                            <td>The collective signature of the current beacon.</td>
+                        </tr>
+                    </table>
+                </details>
+                <details open="true">
+                    <summary>TypeCollectiveBeacon</summary>
+                    <blockquote>
+                        Defines payload data for CollectiveBeacon payload type.
+                    </blockquote>        
+                    <table>
+                        <tr>
+                            <td><b>Name</b></td>
+                            <td><b>Type</b></td>
+                            <td><b>Description</b></td>
+                        </tr>
+                        <tr>
+                            <td>Round</td>
+                            <td>uint64</td>
+                            <td>The round of the current beacon.</td>
+                        </tr>
+                        <tr>
+                            <td>PrevSignature</td>
+                            <td>ByteArray[96]</td>
+                            <td>The collective signature of the previous beacon.</td>
+                        </tr>
+                        <tr>
+                            <td>Signature</td>
+                            <td>ByteArray[96]</td>
+                            <td>The collective signature of the current beacon.</td>
+                        </tr>
+                        <tr>
+                            <td>DistributedPK</td>
+                            <td>ByteArray[48]</td>
+                            <td>The distributed public key.</td>
+                        </tr>
+                    </table>
+                </details>
+            </td>
+        </tr>
     </tr>
 </table>
```

Comment:
```suggestion
    <tr>
        <td>Size</td>
        <td>uint32</td>
        <td>The size of the payload.</td>
    </tr>
    <tr>
        <td>Payload Type</td>
        <td>uint32</td>
        <td>Set to <strong>6</strong> to denote a <i>Collective Beacon</i> payload or to <strong>5</strong> for <i>Beacon</i> payload.</td>
    </tr>
    <tr>
        <td>Version</td>
        <td>uint8</td>
        <td>The version of the payload.</td>
    </tr>
    <tr>
        <td>InstanceID</td>
        <td>uint32</td>
        <td>The identifier of the dRNG instance.</td>
    </tr>
    <tr>
        <td valign="top">dRNG subpayload <code>oneOf</code></td>
        <td colspan="2">
            <details open="true">
            <summary>TypeBeacon</summary>
                <blockquote>
                    Defines payload data for Beacon payload type.
                </blockquote>
                <table>
                    <tr>
                        <td><b>Name</b></td>
                        <td><b>Type</b></td>
                        <td><b>Description</b></td>
                    </tr>
                    <tr>
                        <td>Round</td>
                        <td>uint64</td>
                        <td>The round of the current beacon.</td>
                    </tr>
                    <tr>
                        <td>PartialPK</td>
                        <td>ByteArray[96]</td>
                        <td>The public key of the issuer.</td>
                    </tr>
                    <tr>
                        <td>PartialSignature</td>
                        <td>ByteArray[96]</td>
                        <td>The collective signature of the current beacon.</td>
                    </tr>
                </table>
            </details>
            <details open="true">
                <summary>TypeCollectiveBeacon</summary>
                <blockquote>
                    Defines payload data for CollectiveBeacon payload type.
                </blockquote>        
                <table>
                    <tr>
                        <td><b>Name</b></td>
                        <td><b>Type</b></td>
                        <td><b>Description</b></td>
                    </tr>
                    <tr>
                        <td>Round</td>
                        <td>uint64</td>
                        <td>The round of the current beacon.</td>
                    </tr>
                    <tr>
                        <td>PrevSignature</td>
                        <td>ByteArray[96]</td>
                        <td>The collective signature of the previous beacon.</td>
                    </tr>
                    <tr>
                        <td>Signature</td>
                        <td>ByteArray[96]</td>
                        <td>The collective signature of the current beacon.</td>
                    </tr>
                    <tr>
                        <td>DistributedPK</td>
                        <td>ByteArray[48]</td>
                        <td>The distributed public key.</td>
                    </tr>
                </table>
            </details>
        </td>
    </tr>
</table>
```

Reply:
Still broken for me. Only with this changes it works for me. Github suggestion is a mess. So not sure if the right stuff was copied over in the above suggestion 😅